packages/lib-classifier/src/plugins/drawingTools/experimental/models/marks/FreehandLine/FreehandLine.js

```diff
@@ -201,17 +225,50 @@ const FreehandLineModel = types
     },
 
     splice(startIndex, endIndex) {
-      const deleteCount = endIndex - startIndex - 1
-      const clippedPoints = self.points.slice(startIndex, endIndex + 1)
+      let firstPoint = Math.min(startIndex, endIndex)
+      let lastPoint = Math.max(startIndex, endIndex)
+      /*
+        Reverse the path, if necessary, so that the first point clicked
+        is always the draggable point.
+      */
+      if (firstPoint === endIndex) {
+        const lastIndex = self.points.length - 1
+        self.points.reverse()
+        firstPoint = lastIndex - startIndex
+        lastPoint = lastIndex - endIndex
+      }
+      const deleteCount = lastPoint - firstPoint - 1
+      const clippedPoints = self.points.slice(firstPoint, lastPoint + 1)
       self.clipPath = clippedPoints.map(({ x, y }) => ({ x, y }))
-      self.points.splice(startIndex + 1, deleteCount)
+      self.points.splice(firstPoint + 1, deleteCount)
```

Comment:
[minor] Note to self: don't worry about the _extremely unlikely_ scenario of lastPoint === firstPoint. If deleteCount is -1, the splice functions similarly to deleteCount = 0 and won't throw an exception.